src/quickstart/install-cli-tools.md

```diff
@@ -25,6 +30,16 @@ Install the latest `helmc` cli for Linux or Mac OS X with:
 
     $ curl -sSL https://get.helm.sh | bash
 
+> **NOTICE:** The `unzip` package is a requirement for this command
+
+The installer places the `helmc` binary in your current directory, but you
+should move it somewhere in your $PATH:
+
+    $ ln -fs $PWD/helmc /usr/local/bin/helmc
+
+> **NOTE:** If you are using Linux you may have to prepend this command
```

Comment:
@bacongobbler you're right, the style won't reflect any special formatting or demand user's attention and that is why I've used `>` notation as it will look very close to the example below:

> **NOTE:** This is an example note